internal/helmdeployer/deployer_test.go

```diff
@@ -60,3 +70,90 @@ func TestValuesFrom(t *testing.T) {
 	totalValues = mergeValues(totalValues, configMapValues)
 	a.Equal(expected, totalValues)
 }
+
+func TestPostRenderer_Run_DeleteCRDs(t *testing.T) {
+	tests := map[string]struct {
+		crd                 *apiextensionsv1.CustomResourceDefinition
+		opts                v1alpha1.BundleDeploymentOptions
+		expectedAnnotations map[string]string
+	}{
+		"default (no DeleteCRDResources specified)": {
+			crd: &apiextensionsv1.CustomResourceDefinition{
```

Comment:
I think it would make sense to add a non-CRD resource, just to validate that the annotation is _only_ added to CRDs, in this test case and the last one. How do you see it?

Reply:
yes, that makes sense. Done in https://github.com/rancher/fleet/pull/2024/commits/8ad36f5ad80c5f88be6ee033239b04b7d1985e5c Thanks :)